assets/sass/patterns/organisms/content-header.scss

```diff
@@ -64,41 +66,42 @@
     }
 
     @media only all and (min-width: #{get-rem-from-px($bkpt-site--wide)}em) {
-      height: #{$content-header-padding * 7}px;
+      height: #{$blg-measure-extra-extra-large-in-px}px;
     }
   }
 }
 
 .content-header__body {
-  @include margin($content-header-padding * 2, "top");
-  @include margin($content-header-padding, "bottom");
+  @include blg-spacing("top", "medium", "margin");
+  @include blg-spacing("bottom", "small", "margin");
 
   .content-header--header & {
-    @include margin($content-header-padding * 1.5 + 11 * 2.1818181818, "top"); // 11 * 2.1818181818 is the height of the subject line
+    @include blg-spacing("top", "mediumish", "margin");
     @media only all and (min-width: #{get-rem-from-px($bkpt-site--medium)}em) {
-      @include margin($content-header-padding * 2 + 11 * 2.1818181818, "top");
+      @include blg-spacing("top", "large", "margin");
     }
   }
 }
 
 .content-header__title {
```

Comment:
Reconcile the font setting in this selector with the [helper functions](https://github.com/elifesciences/pattern-library/pull/786/files#diff-4d96201bef5d54fd5b283e903d05e73bR3) for the equivalent task in `typographical-hierarchy.scss`.